packages/ra-ui-materialui/src/field/RichTextField.tsx

```diff
@@ -32,7 +48,11 @@ export const RichTextField: FC<RichTextFieldProps> = memo<RichTextFieldProps>(
                 ) : stripTags ? (
                     removeTags(value)
                 ) : (
-                    <span dangerouslySetInnerHTML={{ __html: value }} />
+                    <span
+                        dangerouslySetInnerHTML={{
+                            __html: purify.sanitize(value),
```

Comment:
That's neat :)

But how about `RichTextInput`? Shouldn't we also sanitize the value we fetch from the server before feeding it to `RichTextInput`?

Reply:
`<RichTextInput>` doesn't use `dangerouslySetInnerHTML`.

Reply:
And [tiptap should no longer be vulnerable](https://security.snyk.io/vuln/SNYK-JS-TIPTAP-575143). So that's fine by me!